_includes/faq/planned-maintenance.md

```diff
@@ -1,13 +1,21 @@
-By default, if a node stays offline for more than 5 minutes, the cluster will consider it dead and will rebalance its data to other nodes. Before temporarily stopping nodes for planned maintenance (e.g., upgrading system software), if you expect any nodes to be offline for longer than 5 minutes, you can prevent the cluster from unnecessarily rebalancing data off the nodes by increasing the `server.time_until_store_dead` [cluster setting](cluster-settings.html) to match the estimated maintenance window.
+By default, if a node stays offline for more than 5 minutes, the cluster will consider it dead and will rebalance its data to other nodes. Before temporarily stopping nodes for planned maintenance (e.g., upgrading system software), if you expect any nodes to be offline for longer than 5 minutes, you can prevent the cluster from unnecessarily rebalancing data off the nodes by increasing the `server.time_until_store_dead` [cluster setting](cluster-settings.html) to match the estimated maintenance window. In addition, to ensure that load balancers won't send client traffic to a node about to be shut down, you can increase the `server.shutdown.drain_wait` setting.
```

Comment:
It's weird to me to mix the `drain_wait` setting in so closely with the `time_until_store_dead` setting, since they're not very related. `drain_wait` is important any time you're stopping a cockroach process, even if it's only going to be a few seconds before you restart it, while `time_until_store_dead` is only relevant if a node is going to be offline for more than 5 minutes.

We could give this a separate subsection, maybe? Or just fully explain one setting before mentioning the other rather than blending them?

Also, this is less clear cut, but you could argue that `drain_wait` is relevant for more than just "planned maintenance" situations since our docs up to this point seem to have defined "planned maintenance" to mean events longer than 5 minutes.

Reply:
Thanks, @a-robinson. I made some changes. PTAL.